### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by constantinfite** [ultralytics/yolov5#1786]
## ❔Question
What is the graph in TensorBoard to look at to see the loss function of the model ?

Is one of these ?
![image](https://user-images.githubusercontent.com/57963890/103159640-208dda80-47cc-11eb-8a24-1987771657d8.png)

**Comment by glenn-jocher** [comment]
@constantinfite yes, these are the 3 components in the YOLO multi-component loss function.

**Comment by constantinfite** [comment]
@glenn-jocher thanks, I also wonder why my obj_loss is not varying (neither up nor down)
What number of epoch is optimal when the loss doesn't change ?
Like this 
![image](https://user-images.githubusercontent.com/57963890/103160707-91d38a80-47d8-11eb-8259-e0c29eb24e87.png)

**Comment by glenn-jocher** [comment]
@constantinfite train until you overfit. Upon training completion best.pt will contain the weights that produced the best fitness.

**Comment by constantinfite** [comment]
@glenn-jocher ok thanks ! I will try to train on more epoch

**Comment by constantinfite** [comment]
@glenn-jocher could you tell me what obj_loss stand for ? It is the loss for what ?

**Comment by glenn-jocher** [comment]
@constantinfite objectness loss
